.github/workflows/serverless-binary-size.yml

```diff
@@ -0,0 +1,100 @@
+name: "Serverless Binary Size"
+
+on:
+  pull_request:
+
+env:
+  SIZE_ALLOWANCE: 100000  # 0.1 MB
+
+jobs:
+  comment:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout datadog-agent repository
+        uses: actions/checkout@v3
+        with:
+          path: go/src/github.com/DataDog/datadog-agent
+
+      - name: Checkout the datadog-lambda-extension repository
+        uses: actions/checkout@v3
+        with:
+          repository: DataDog/datadog-lambda-extension
+          path: go/src/github.com/DataDog/datadog-lambda-extension
+          ref: rey.abolofia/viz-tools
```

Comment:
TODO: This PR should not be merged until after we merge https://github.com/DataDog/datadog-lambda-extension/pull/122